test/exceptions/src/assert_fail.d

```diff
@@ -158,6 +163,20 @@ void testTemporary()
     assert(Bad() == Bad());
```

Comment:
Thanks! Would you mind adding a similar output test for `Bad` as well.

Reply:
Can do, but its another compilable test (as `struct`formatting is already covered above)